packages/cozy-client/src/models/account.js

```diff
@@ -114,3 +116,22 @@ export const buildAccount = (konnector, authData) => {
     state: null
   }
 }
+
+/**
+ * Look if the given account is connected or not. Does a trigger which is referencing this account
+ * exist
+ *
+ * @param {CozyClient} client - CozyClient instance
+ * @param {String} accountId - id of the account
+ * @returns {Promise<Boolean>}
+ */
+export const isAccountConnected = async (client, accountId) => {
```

Comment:
I'd prefer you to pass the entire account object. Not just the id. 

Cozy-Client's models are about a "functional object", not just a subpart. It's easier to remember that all the methods from models take the object